deployment/terraform/modules/vpc/main.tf

```diff
@@ -2,31 +2,26 @@ resource "aws_vpc" "vpc" {
   cidr_block           = "${var.vpc_cidr_block}"
   enable_dns_support   = true
   enable_dns_hostnames = true
-
-  tags {
-    Name = "${terraform.workspace} (terraform)"
-  }
+  tags                 = "${merge(var.tags, map("Name", terraform.workspace))}"
 }
 
 resource "aws_subnet" "public_subnet" {
   vpc_id                  = "${aws_vpc.vpc.id}"
   cidr_block              = "${var.public_cidr_block}"
   availability_zone       = "${var.availability_zone_a}"
   map_public_ip_on_launch = true
-
-  tags {
-    Name = "${terraform.workspace} (terraform public subnet)"
-  }
+  tags              = "${merge(
```

Comment:
It's been a while, but I think you prefer the "=" to be aligned in each section, and I think there's a tool that does that for you?

Reply:
Pushed a fix. HCL plugin for PyCharm does have formatting functionality but it doesn't work very well.